spec/rubocop/cop/style/multiline_operation_indentation_spec.rb

```diff
@@ -168,6 +168,15 @@
       expect(cop.messages).to eq(['Use 2 (not 6) spaces for indenting an ' \
                                   'expression spanning multiple lines.'])
     end
+
+    it 'registers an offense for aligned code on LHS of assignment' do
```

Comment:
I think it might be good to add one example without an assignment as well, just for clarity's sake.